tests/testthat/_snaps/shuffle_ard.md

```diff
@@ -407,108 +55,50 @@
 # shuffle_ard fills missing group levels if the group is meaningful
 
     Code
-      shuffle_ard(bind_ard(ard_continuous(ADSL, by = "ARM", variables = "AGE",
-        statistic = ~ continuous_summary_fns("mean")), dplyr::tibble(group1 = "ARM",
-        variable = "AGE", stat_name = "p", stat_label = "p", stat = list(0.05))))
+      shuffle_ard(dplyr::filter(bind_ard(ard_continuous(ADSL, by = "ARM", variables = "AGE", statistic = ~ continuous_summary_fns("mean")), dplyr::tibble(group1 = "ARM", variable = "AGE", stat_name = "p",
+        stat_label = "p", stat = list(0.05))), dplyr::row_number() <= 5L))
     Output
       # A tibble: 4 x 5
         ARM                  variable context    stat_name  stat
         <chr>                <chr>    <chr>      <chr>     <dbl>
       1 Placebo              AGE      continuous mean      75.2 
       2 Xanomeline High Dose AGE      continuous mean      74.4 
       3 Xanomeline Low Dose  AGE      continuous mean      75.7 
-      4 Overall ARM          AGE      <NA>       p          0.05
+      4 <NA>                 AGE      <NA>       p          0.05
```

Comment:
This seems like an important difference that we need to address @bzkrouse ! Do you know why it's different?

Reply:
@ddsjoberg This is because the rename_ard_columns() used to "fill" all groups/variables that had a value with a missing level. in this case, group1 = "ARM" but there was no group1_level (similar to result of `ard_stats_t_test`) so it was filled.

Reply:
In this case, the first three rows are the mean of age by the three treatment arms. The last row is a statistic that is only related to AGE (and not ARM, created with `dplyr::tibble(group1 = "ARM", variable = "AGE", stat_name = "p", stat_label = "p", stat = list(0.05))`).

Since this result has nothing to do with ARM, the ARM column isn't filled with anything. This kind of filling is where I would see shuffle would take over, as it is filling in `ARM = 'Overall'` for summary statistics that were calculated with no involvement of ARM.

Reply:
okay this makes sense. Shuffle will need to handle this before the rename function then!